projects/packages/forms/src/blocks/contact-form/class-contact-form-block.php

```diff
@@ -294,20 +294,22 @@ public static function load_editor_scripts() {
 		$dashboard_view_switch   = new Dashboard_View_Switch();
 		$contact_form            = new Contact_Form( array() );
 		$defaults                = $contact_form->defaults;
-		$admin_url               = $dashboard_view_switch->get_forms_admin_url( 'spam' );
+		$form_responses_url      = $dashboard_view_switch->get_forms_admin_url();
+		$form_responses_spam_url = $dashboard_view_switch->get_forms_admin_url( 'spam' );
 		$akismet_active_with_key = Jetpack::is_akismet_active();
 		$akismet_key_url         = admin_url( 'admin.php?page=akismet-key-config' );
 		$preferred_view          = $dashboard_view_switch->get_preferred_view();
 
 		$data = array(
 			'defaults' => array(
-				'to'                   => $defaults['to'],
-				'subject'              => $defaults['subject'],
-				'formsAdminUrl'        => $admin_url,
-				'akismetActiveWithKey' => $akismet_active_with_key,
-				'akismetUrl'           => $akismet_key_url,
-				'assetsUrl'            => Jetpack_Forms::assets_url(),
-				'preferredView'        => $preferred_view,
+				'to'                    => $defaults['to'],
+				'subject'               => $defaults['subject'],
+				'formsResponsesUrl'     => $form_responses_url,
+				'formsResponsesSpamUrl' => $form_responses_spam_url,
+				'akismetActiveWithKey'  => $akismet_active_with_key,
+				'akismetUrl'            => $akismet_key_url,
+				'assetsUrl'             => Jetpack_Forms::assets_url(),
+				'preferredView'         => $preferred_view,
```

Comment:
This file originally had $admin_url / formsAdminUrl, but both were linking to spam responses and used by the Akismet card.

We now have $form_responses_url / formsResponsesUrl (for the main inbox) and $form_responses_spam_url / formsResponsesSpamUrl (for the spam tab).

None of the other values being returned here changed - there's just a spacing change since  formsResponsesSpamUrl is long and the => need to be aligned.